sphinx/writers/latex.py

```diff
@@ -1886,9 +1891,10 @@ def warner(msg):
             # get consistent trailer
             hlcode = hlcode.rstrip()[:-14]  # strip \end{Verbatim}
             self.body.append('\n' + hlcode + '\\end{sphinxVerbatim%s}\n' %
-                             (self.table and 'intable' or ''))
+                             ((self.table and not self.in_footnote) and
```

Comment:
I feel this condition become complex. It's a bit unreadable. So could you rewrite this to simple if-else notation?